### ISSUE TITLE
Sliding I’d tabs during drag drop 

### ISSUE THREAD

**Issue by Alexey-T** [Alexey-T/ATTabs#57]
@srd-software Maybe if someone makes big donation I could add it, not sure... I’d want $300 :)

**Comment by srd-software** [comment]
Yeah, it would be a big revision I'm sure. At least researching that I came up with some code that will help me alot in my other projects so it was definitely worth doing.

Current tabs are looking great.
